user/bfs/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "bfs"
+pkgver = "4.0.6"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--enable-release",
+    "--with-libacl",
+    "--with-libcap",
+    "--with-liburing",
+    "--with-oniguruma",
+]
+# the integration-tests fail to run with cbuild
+make_check_target = "unit-tests"
+hostmakedepends = ["pkgconf"]
+makedepends = ["acl-devel", "libcap-devel", "liburing-devel", "oniguruma-devel"]
+checkdepends = ["bash"]
+pkgdesc = "Breadth-first version of the UNIX find command"
+license = "0BSD"
+url = "https://github.com/tavianator/bfs"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "446a0a1a5bcbf8d026aab2b0f70f3d99c08e5fe18d3c564a8b7d9acde0792112"
+hardening = ["cfi", "vis"]
+
+
+def post_install(self):
+    for f in (self.cwd / "docs").glob("*.md"):
```

Comment:
we don't install docs

Reply:
The doc files are picked up automatically by the `bfs-doc` subpackage. Should I not include those files in the `-doc` subpackage?

Reply:
changes made after discussion in IRC/matrix chat room